e2e/helm.e2e.spec.ts

```diff
@@ -0,0 +1,96 @@
+import path from 'path';
+import {
+  ElectronApplication, BrowserContext, _electron, Page, Locator
+} from 'playwright';
+import { test, expect } from '@playwright/test';
+import {
+  createDefaultSettings, kubectl, helm, tearDownHelm, playwrightReportAssets
+} from './utils/TestUtils';
+
+let page: Page;
+
+test.describe.serial('Helm Deployment Test', () => {
+  let mainTitle: Locator;
+  let electronApp: ElectronApplication;
+  let context: BrowserContext;
+
+  const mainTitleSelector = '[data-test="mainTitle"]';
+
+  test.beforeAll(async() => {
+    createDefaultSettings();
+
+    electronApp = await _electron.launch({
+      args: [
+        path.join(__dirname, '../'),
+        '--disable-gpu',
+        '--whitelisted-ips=',
+        '--disable-dev-shm-usage',
+      ]
+    });
+    context = electronApp.context();
+
+    await context.tracing.start({ screenshots: true, snapshots: true });
+    page = await electronApp.firstWindow();
+  });
+
+  test.afterAll(async() => {
+    await context.tracing.stop({ path: playwrightReportAssets(path.basename(__filename)) });
+    await tearDownHelm();
```

Comment:
The `test.afterAll()` will run after all tests included into the group `describe`. This function will be triggered on both statuses: PASS or FAIL.
Comparing with `jest`, jest will try to kill the process (if it wasn't defined a flow inside of the test spec). On `playwright`, doesn't matter if the test case will pass or fail, the `afterAll()` function will be triggered anyway. We don't need to worry how we handle the teardown helper, as long it triggers before `electronApp.close()`.

Reply:
But we still need to be careful to always trigger `electronApp.close()` even if `tearDownHelm` encounters an issue and throws, right?

Reply:
Hmm..
if an exception throws from the helm command, it will, probably, stop the test framework before trigger ElectronApp.close...

Doesn't hurt add another afterAll(tearDown())... just in case to face the described scenario. I'll fix that. Thanks for the brainstorming :)